clickhouse/columns/nullable.h

```diff
@@ -42,10 +42,41 @@ class ColumnNullable : public Column {
 
     /// Makes slice of the current column.
     ColumnRef Slice(size_t begin, size_t len) override;
+    void Swap(Column&) override;
+
+    ItemView GetItem(size_t) const override;
 
 private:
     ColumnRef nested_;
     std::shared_ptr<ColumnUInt8> nulls_;
 };
 
+
+inline const Column & removeNullable(const Column & col)
+{
+    if (col.Type()->GetCode() == Type::Nullable)
+    {
+        return *static_cast<const ColumnNullable &>(col).Nested();
+    }
+
+    return col;
+}
+
+inline Column & removeNullable(Column & col)
+{
+    return const_cast<Column &>(removeNullable(const_cast<const Column&>(col)));
```

Comment:
Why `const_cast` if you have an overload for both const and mutable refs? Moreover, maybe better to provide a non-const `Nested()` and don't use `const_cast` at all?

Reply:
removed that altogether, it is not used anywhere